charts/intel-resource-drivers/README.md

```diff
@@ -9,37 +9,34 @@ helm repo update
 You can execute `helm search repo intel` command to see pulled charts [optional].
 
 ## Install Helm Chart
-CRDs of the GPU driver are installed as part of the chart first.
-
 ```
-helm install intel-gpu-resource-driver intel/intel-gpu-resource-driver \
---create-namespace --namespace intel-gpu-resource-driver
+helm install intel-resource-drivers intel/intel-resource-drivers
 ```
 ## Upgrade Chart
 ```
-helm upgrade intel-gpu-resource-driver intel/intel-gpu-resource-driver [flags]
+helm upgrade intel-resource-drivers intel/intel-resource-drivers [flags]
 ```
 
 ## Uninstall Chart
 ```
-helm uninstall intel-gpu-resource-driver --namespace intel-gpu-resource-driver
+helm uninstall intel-resource-drivers
 ```
 
 ## Configuration
 See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments:
 
 ```console
-helm show values intel/intel-gpu-resource-driver
+helm show values intel/intel-resource-drivers
 ```
 
 You may also run `helm show values` on this chart's dependencies for additional options.
 
 | Key | Type | Default |
 |-----|------|---------|
 | image.repository | string | `intel` |
-| image.name | string | `"intel-gpu-resource-driver"` |
+| image.name | string | `"intel-resource-drivers"` |
 | image.pullPolicy | string | `"IfNotPresent"` |
-| image.tag | string | `"v0.4.0"` |
+| image.tag | string | `"v0.5.1"` |
```

Comment:
The helm chart appears to have different images now, is this `value` still in use? Please check the rest of `values` in this table too.